oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/StorageEsInstaller.java

```diff
@@ -104,8 +111,14 @@ public boolean isExists(Model model) throws StorageException {
             structures.putStructure(
                 tableName, template.get().getMappings(), template.get().getSettings()
             );
-            exist = structures.containsMapping(tableName, createMapping(model))
-                && structures.compareIndexSetting(tableName, createSetting(model));
+            boolean containsMapping = structures.containsMapping(tableName, createMapping(model));
+            // "no-init mode" no needs to check index settings for updating,
+            // to avoid conflicts between "init mode and no-init mode" index settings configurations
```

Comment:
```suggestion
            // Do not check index settings in the "no-init mode",
            // because the no-init mode OAP server doesn't take responsibility for index settings.
```